invenio_records_resources/services/custom_fields/validate.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Custom Fields for InvenioRDM."""
+
+from .errors import CustomFieldsNotConfigured, InvalidCustomFieldsNamespace
+
+
+def validate_custom_fields(available_fields, namespaces, given_fields=None):
```

Comment:
maybe we add a simple test for this function.

Reply:
added.